docs/monitoring/metrics.md

```diff
@@ -1071,6 +1072,120 @@ Thus, in order to infer the metric identifier:
   </tbody>
 </table>
 
+### NettyShuffle
```

Comment:
do users actually know what NettyShuffle is?

Reply:
True users do not know about `NettyShuffle`, we will still have only one  at the moment.
It could be `Default network shuffle` then.